src/templates/member-profile.tsx

```diff
@@ -1,14 +1,111 @@
 import * as React from 'react';
-import { MainLayout } from '../layout/main.js'; 
-
-type Props = {
+import Marquee from '../components/marquee.js';
+import Status from '../components/status.js';
+type Friend = {
+  name: string;
+  alt: string;
+  image: string;
+}
+export type MemberProps = {
   name: string;
+  pronouns?: 'he' | 'she' | 'they' | string;
+  /** years of experience */
+  yoe?: number;
+  /** url for gravatar or profile pic */
+  avatar?: string;
+  bio?: string;
+  website?: string;
+  githubHandle?: string;
+  status?: string;
+  interests?: string[];
+  music?: string[];
+  heroes?: string[];
+  books?: string[];
+  children?: JSX.Element;
+  friends: Friend[];
 }
 
-export function MemberProfile(props: Props) {
+export function MemberProfile(props: MemberProps) {
+  const { name, pronouns, yoe, avatar, bio, website, githubHandle, status, interests, music, heroes, books, friends } = props;
+  const githubUrl = `https://github.com/${githubHandle}`;
+
+  return (
+    <>
+      <html>
```

Comment:
why did you delete the layout?

Reply:
Extreme separation of concerns initially 😆